config/lib/app.js

```diff
@@ -11,6 +11,11 @@ var config = require('../config'),
 // Initialize Models
 mongoose.loadModels();
 
+// Populate DB with sample data
+if (config.seedDB) {
+	require('./seed');
```

Comment:
This is super annoying... It is 2 spaces on my editor but committing as 4 spaces.  Not sure what to do:

![image](https://cloud.githubusercontent.com/assets/3578259/9225473/3e6b212a-40cf-11e5-96e4-1c035c884b5c.png)